lib/Sema/TypeCheckAccess.cpp

```diff
@@ -1660,21 +1665,27 @@ class ExportabilityChecker : public DeclVisitor<ExportabilityChecker> {
     return true;
   }
 
+  void checkExtensionMemberIfNeeded(const ValueDecl *D) {
+    auto *extension = dyn_cast<ExtensionDecl>(D->getDeclContext());
```

Comment:
It's kind of odd that this is called for every member but it only checks and diagnoses the extension itself?

Reply:
Maybe the name is wrong? The trouble is we only want to diagnose the extension if one of the members is public.

Reply:
(Actually, strictly speaking I'd like this to be a diagnostic on the member somehow even though the problem is at the extension level, but that can come when we productize this thing.)